translations/dde-store_zh_CN.ts

```diff
@@ -0,0 +1,435 @@
+        <source>PackageKit Error</source>
+        <translation>包工具出错</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../backend/settings.cpp" line="85"/>
+        <source>Basic</source>
+        <translation>基础设置</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="86"/>
+        <source>View</source>
+        <translation>视图</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="87"/>
+        <source>Behaviour</source>
+        <translation>行为</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="88"/>
+        <source>Notifications</source>
+        <translation>通知</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="89"/>
+        <source>Show non-app updates as</source>
+        <translation>没有软件更新时显示</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="90"/>
+        <source>Max apps per list</source>
+        <translation>列表最大软件数</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="91"/>
+        <source>Minimize to tray on exit</source>
+        <translation>退出时最小化到托盘</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="92"/>
+        <source>Check for updates</source>
+        <translation>检查更新频率</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="93"/>
+        <source>App installed</source>
+        <translation>安装完软件</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="94"/>
+        <source>App uninstalled</source>
+        <translation>卸载完软件</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="95"/>
+        <source>Available updates</source>
+        <translation>更新就绪</translation>
+    </message>
+    <message>
+        <location filename="../backend/settings.cpp" line="96"/>
+        <source>Finished updates</source>
+        <translation>更新完成</translation>
+    </message>
+</context>
+<context>
+    <name>SnapPlugin</name>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="112"/>
+        <source>Installed &quot;%1&quot;</source>
+        <translation>已安装 %1</translation>
+    </message>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="150"/>
+        <source>Uninstalled &quot;%1&quot;</source>
+        <translation>已卸载 %1</translation>
+    </message>
+    <message>
+        <location filename="../backend/snapplugin.cpp" line="218"/>
+        <source>Classic mode required</source>
+        <translation type="unfinished"></translation>
```

Comment:
A couple strings are still untranslated. Did you have issues with those ones?

Reply:
[I don't know where this sentence appears, and what it does.](https://translate.google.cn/?sl=auto&tl=en&text=%E6%88%91%E4%B8%8D%E4%BA%86%E8%A7%A3%E8%BF%99%E5%8F%A5%E8%AF%9D%E5%87%BA%E7%8E%B0%E5%9C%A8%E5%93%AA%E9%87%8C%EF%BC%8C%E5%AE%83%E7%9A%84%E4%BD%9C%E7%94%A8%E6%98%AF%E4%BB%80%E4%B9%88%E3%80%82&op=translate)

Reply:
Some Snap applications need to be installed in "classic" mode. Without it, Snap applications are isolated from the rest of the system, so some can't function properly.